contracts/gateway.fc

```diff
@@ -25,9 +26,13 @@ const op::authority::update_tss = 202;
 const op::authority::update_code = 203;
 const op::authority::update_authority = 204;
 
-;; GAS FEE =========================================
-;; Let's use const for now and refine this later
-const deposit_gas_fee = 10000000; ;; 0.01 TON
+;; GAS FEES =========================================
+;; These constants are empirically determined based on tests
+;; and slighly bumped to cover all other non-gas fees (e.g. storage, fwd_fee, etc.)
```

Comment:
What slighly bumped means in this context? I think we should set a comfortable buffer to ensure we don't have overpayment issue

Reply:
tx fee = gas_fee + additional fees (storage fee, forward fee, etc...). Based on the test cases I made tx fee be approximated to `gas_price(gas::operation_abc)`. 

This allows each operation to have a cost "ceiling" i.e. `real total fees` <= `gas_price(gas::operation_abc)`